packages/api/src/objectSet/ObjectSetListener.ts

```diff
@@ -55,3 +55,19 @@ type ObjectUpdate<
   object: Osdk.Instance<O, never, P>;
   state: "ADDED_OR_UPDATED" | "REMOVED";
 };
+
+/**
+ * Options for subscribing to an ObjectSet.
+ *
+ * @property properties - The properties to request a subscription for. Requesting specific properties limits the possible properties
+ * that can be returned from the subscription. If not provided, all properties will be requested and potentially be returned on updates.
```

Comment:
Lets update this comment to make it clear that empty array will _also_ give you reference updates (like geotime)